common/src/main/java/org/opensearch/ml/common/input/nlp/TextDocsMLInput.java

```diff
@@ -114,7 +114,11 @@ public TextDocsMLInput(XContentParser parser, FunctionName functionName) throws
                 case TEXT_DOCS_FIELD:
                     ensureExpectedToken(XContentParser.Token.START_ARRAY, parser.currentToken(), parser);
                     while (parser.nextToken() != XContentParser.Token.END_ARRAY) {
-                        docs.add(parser.text());
+                        if (parser.currentToken() == null || parser.currentToken() == XContentParser.Token.VALUE_NULL) {
```

Comment:
What happens if the input is `[null, null]`?

Reply:
The depends on the model. If model can't accept, it will throw exception.